client/app/css/less/ovh-ui-kit/icon.less

```diff
@@ -4,4 +4,8 @@
     &.oui-icon_small {
         font-size: 1.4rem;
     }
+
+    &.oui-icon_info-medium {
```

Comment:
What's about having this directly on ovh-ux/ovh-ui-kit instead?

Reply:
We have to discuss to have css helpers for colors (`background-color` / `color`).